ocaml/libs/tracing/tracing.ml

```diff
@@ -684,9 +684,9 @@ module Export = struct
           let headers =
             Cohttp.Header.of_list
               [
-                ("accepts", "application/json")
-              ; ("content-type", "application/json")
+                ("content-type", "application/json")
               ; ("content-length", string_of_int (String.length body))
+              ; ("host", "jaeger.xenrt.citrite.net:9411")
```

Comment:
should we use the capitalised version here? Although it should be case insensitive I think you found an article where some proxies are buggy